extra/release_tool.py

```diff
@@ -1576,12 +1588,47 @@ def trigger_build(state, tag_avail):
 
         reply = ask("Will trigger a build with these values, ok? (no) ")
         if reply.startswith("Y") or reply.startswith("y"):
-            break
+            if USE_GITLAB:
+                trigger_gitlab_build(params, extra_buildparams)
+            else:
+                trigger_jenkins_build(params, extra_buildparams)
+            return
 
         reply = ask(
-            "Do you want to change any of the parameters (Y/N/open in Editor)? "
+            """Do you want to change any of the parameters?
+Y = Yes
+N = No (Quit)
+E = Open in editor
+R = Reset all to default
+ET = Enable all tests
+DT = Disable all tests
+EP = Enable all platforms
+DP = Disable all platforms
+ER = Enable publishing of the release
+DR = Disable publishing of the release
```

Comment:
I would be explicit here adding "automatic" somewhere. The publishing of the release is always enabled, just manually done.
```suggestion
ER = Enable automatic publishing of the release
DR = Disable automatic publishing of the release
```